### PR TITLE
Update support for decrypting autoforwarded messages

### PR DESCRIPTION
**Pull request by larabr**
Follow up to #1 : update implementation to reflect spec changes to KDF params (v2 -> v255) and new forwarding-related key flag (0x40).

### CODE REVIEW

File: src/type/kdf_params.js [hunk 1]
```diff
@@ -27,34 +27,38 @@
...
-      this.replacementKDFParams = null;
+    }
+  }
+
+  getReplacementParams() {
+    if (this.version === VERSION_FORWARDING) {
+      return util.concatUint8Array([
+        new Uint8Array([this.hash, this.cipher]),
+        this.replacementFingerprint
+      ]);
```
Comment (twiss) [inline src/type/kdf_params.js]: Is this function needed? I don't see where it's used

Comment (larabr) [reply src/type/kdf_params.js]: oops, yeah it's a leftover, from before I thought of reusing `write`

File: openpgp.d.ts [hunk 2]
```diff
@@ -903,3 +904,14 @@ export namespace enums {
...
+interface KDFParamsData {
+  version: number;
+  hash: enums.hash;
+  cipher: enums.symmetric;
+  replacementFingerprint?: Uint8Array;
+}
+export class KDFParams {
+  constructor(data: KDFParamsData);
+  write(forReplacementParams?: boolean): Uint8Array;
+}
```
Comment (twiss) [inline openpgp.d.ts]: (missing newline here, and maybe add one after KDFParamsData too)

**Review by twiss — APPROVED** [review]
One nitpick below, other than that LGTM :+1:

**Pull request merged**